src/Service/Logger.php

```diff
@@ -95,24 +97,59 @@ private function log(string $type, int $timestamp, string $line, bool $logUser =
       $line .= sprintf(' Remote ip: %s', $ip_address);
     }
 
+    $config = $this->configFactory->get(SettingsForm::$configName);
+    $plugin_id = $config->get('provider') ?? SettingsForm::OS2WEB_AUDIT_DEFUALT_PROVIDER;
+
+    $queueStorage = $this->entityTypeManager->getStorage('advancedqueue_queue');
+    /** @var \Drupal\advancedqueue\Entity\Queue $queue */
+    $queue = $queueStorage->load(self::OS2WEB_AUDIT_QUEUE_ID);
+    $job = Job::create(LogMessages::class, [
+      'type' => $type,
+      'timestamp' => $timestamp,
+      'line' => $line,
+      'plugin_id' => $plugin_id,
+      'metadata' => $metadata,
+    ]);
+
+    $queue->enqueueJob($job);
+
+    $logger_context = [
+      'job_id' => $job->getId(),
+      'operation' => 'log message queued',
+    ];
+
+    $this->watchdog->get(self::OS2WEB_AUDIT_LOGGER_CHANNEL)->notice('Added audit logging message to queue for processing.', $logger_context);
```

Comment:
This will also kill the database with not need watchdog messages

Reply:
Reduced watchdog logging to job creation process in 1bf3fcb8f9b17c5da64336d3d35492cec50db2ed.